src/script-runner.js

```diff
@@ -25,10 +26,10 @@ class CodeExecutor extends PluginClient {
   execute (script) {
     if (script) {
       try {
-        (new Function(script))()
-        if (mocha.suite & mocha.suite.suites && mocha.suite.suites.length) {
+        if (mocha.suite && mocha.suite.root) {
```

Comment:
why do you run `mocha.run()` before the script is actually executed?

Reply:
updated